getting-started/Vulnerability-Scan.md

```diff
@@ -0,0 +1,65 @@
+**MICROSFOT CONFIDENTIAL** 
+
+# Microsoft Defender: Vulnerability Pre-Scan Check  
+
+### Overview
+Vulnerability scanning is the process of discovering, analyzing, and reporting on security flaws and vulnerabilities. Vulnerability scans are conducted via automated vulnerability scanning tools to identify potential risk exposures. Here we use Microsoft Defender for Cloud and Qualys to identify vulnerabilities in your containers and use publicly available information from www.cve.org to assist in remediation. You can use other open source tools such as Twist Lock, Clair, and Aqua Security to pre-scan your image, however we use Microsoft Defender to complete the cert process so it is recommended that you use the same tool to ensure consistency. This doc will walk you through a pre-scan process using the Azure Portal/ Defender for your container registry to help speed the Azcert process down the line.  
+
+Before beginning the pre-scan process, you will need to set up a Microsoft Azure Portal account to access Defender, if you don’t have an account already. Access to Defender is a paid resource. If you do not have an Azure portal account, you can create one at [Azure portal](https://portal.azure.com).
+
+### Important Notes to Keep in Mind 
+* CVEs are common vulnerabilities and exposures of publicly disclosed computer security flaws. During the pre-scan process you will use the CVE link to identify the vulnerability and remediate it.  
+* Once your image is scanned, in the pop-up blade you will see a CVSS score and if this cvss 3.0 base score is >7 the image will be blocked until remediated. Below is a chart outlining the severity and score range of vulnerabilities.  
+
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/949fb0ac926aa16df4e3ec421a95a5f4960c7658/getting-started/images/cvss3.png" alt="cvss3" width="500"/>
+
+### Vulnerability Pre-Scan Steps 
+1. Sign into your Microsoft Azure Portal Account  
+
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step1-azureportal.png" alt="step1-azureportal" width="800"/>
+
+2. In the search bar, enter container registries  
+
+ <p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step2-search.png" alt="step2-search" width="800"/>
+
+3. Select your container registry that you have already packaged and uploaded (If you need help uploading the package please refer to here: [Prepare your Azure container technical assets for a Kubernetes application | Microsoft Learn](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=windows) and the additional links at the end of this document). Once selected your view should look something like this: 
```

Comment:
nit: Select your container registry that you have used to upload your application package

Reply:
done